x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importance_summary/feature_importance_summary.tsx

```diff
@@ -150,16 +168,16 @@ export const FeatureImportanceSummaryPanel: FC<FeatureImportanceSummaryPanelProp
     if (isRegressionTotalFeatureImportance(totalFeatureImportance[0])) {
       classificationType = 'regression';
 
-      sortedData = (totalFeatureImportance as RegressionTotalFeatureImportance[]).map(
-        (d: RegressionTotalFeatureImportance) => ({
+      sortedData = (totalFeatureImportance as RegressionTotalFeatureImportance[])
+        .map((d: RegressionTotalFeatureImportance) => ({
           featureName: d.feature_name,
           meanImportance: d.importance.mean_magnitude,
-        })
-      );
+        }))
+        .sort((a, b) => b.meanImportance - a.meanImportance);
     }
 
     // sort from largest importance at top to smallest importance at bottom
-    sortedData = sortedData.sort((a, b) => b.meanImportance - a.meanImportance);
+    // sortedData = sortedData.sort((a, b) => b.meanImportance - a.meanImportance);
```

Comment:
Guess this can be cleaned up, remove the commented code and move the text comment above where the sorting is now happening.

Reply:
Thanks. I removed it here https://github.com/elastic/kibana/pull/78238/commits/b73325bd226c5dddbd573e8b0f6037b33f13c19f.